Client/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -1013,7 +1013,7 @@ bool CLuaPedDefs::SetElementBoneMatrix(lua_State* const luaVM, CClientPed* entit
     return theEntity ? theEntity->SetBoneMatrix(static_cast<eBone>(boneId), boneMatrix) : false;
 }
 
-std::variant<bool, CMatrix> CLuaPedDefs::GetElementBoneMatrix(lua_State* const luaVM, CClientPed* entity, std::int32_t boneId)
+std::variant<bool, std::optional<CMatrix>> CLuaPedDefs::GetElementBoneMatrix(lua_State* const luaVM, CClientPed* entity, std::int32_t boneId, bool doUseTables)
```

Comment:
```suggestion
bool CLuaPedDefs::GetElementBoneMatrix(lua_State* const luaVM, CClientPed* entity, std::int32_t boneId)
```
How about always returning the table instead of CMatrix? I think we should only return CMatrix for oop version of this function.

Reply:
I mean, yeah sure